gazu/client.py

```diff
@@ -129,13 +129,14 @@ def get_full_url(path):
     return url_path_join(get_host(), path)
 
 
-def get(path, json_response=True, params={}):
+def get(path, json_response=True, params=None):
     """
     Run a get request toward given path for configured host.
 
     Returns:
         The request result.
     """
+    params = params or {}
```

Comment:
Why did you change that?

Reply:
Because it is an anti-pattern to initialise dict in function signatures.
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

I can leave it but it may lead to "unforeseen consequences"